packages/hint/src/lib/types/jsdom-async-html.ts

```diff
@@ -134,6 +134,10 @@ export class JSDOMAsyncWindow implements IAsyncWindow {
         return this._document;
     }
 
+    public get dom(): JSDOM | undefined {
```

Comment:
Shouldn't this whole file be able to be deleted since we're only using `JSDOM` indirectly for evaluation purposes now? In the JSDOM connector direct access is available and we don't need an intermediate abstraction layer.